website/docs/components/code-block/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: CodeBlock
+description: A container for displaying formatted chunks of code with syntax highlighting and related features.
+caption: A container for formatting chunks of code with syntax highlighting.
+links:
+  figma: https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?type=design&node-id=44796%3A2786&mode=design&t=pZ5PoxE7Q7U0kDaa-1
+  github: https://github.com/hashicorp/design-system/tree/main/packages/components/addon/components/hds/code-block
+related: ['components/copy/snippet', 'components/copy/button']
+previewImage: assets/illustrations/components/code-block.jpg
+navigation:
+  keywords: ['code', 'snippet', 'copy', 'text', 'editor', 'language', 'example']
```

Comment:
```suggestion
  keywords: ['code', 'snippet', 'copy', 'text', 'editor', 'language', 'example', 'syntax', 'highlight']]
```

Reply:
Yep, I made this change on the other branch. I'll try to reconcile the differences.

Reply:
Resolved in https://github.com/hashicorp/design-system/pull/1749/commits/19a092530be5f2419739f829a8d000a99a7cb4a7